scss/gallery/_media-attachment.scss

```
@@ -160,6 +160,7 @@
   & > .icon {
     height: 44px;
     width: 44px;
+    border-radius: 10%;
```

```suggestion
    border-radius: 7%;
```

i played around a but -
sth. smaller fits better the the rounding of the outer bubble that already have about 10% rounding,